resources/devices/mobile-phone/motorola.json

```diff
@@ -2451,7 +2451,20 @@
             "Device_Name": "Defy Mini",
             "Device_Maker": "Motorola",
             "Device_Brand_Name": "Motorola",
-            "Device_Pointing_Method": "unknown",
```

Comment:
Looked up this device, appears to be a touchscreen device: http://www.gsmarena.com/motorola_defy_mini_xt320-4375.php